safekeeper/src/timeline.rs

```diff
@@ -425,7 +425,7 @@ impl Timeline {
             flush_lsn: Some(shared_state.sk.wal_store.flush_lsn()),
             // note: this value is not flushed to control file yet and can be lost
             commit_lsn: Some(shared_state.sk.inmem.commit_lsn),
-            s3_wal_lsn: Some(shared_state.sk.inmem.s3_wal_lsn),
+            s3_wal_lsn: Some(shared_state.sk.inmem.wal_backup_lsn),
```

Comment:
I'm a bit confused about this rename, why have we decided to rename `s3_wal_lsn` to `wal_backup_lsn`? Both are fine for me by the way.

Reply:
Hm. I think i did that because I saw wal_backup_lsn in @antons-antons PR, but indeed I think both are ok. Let's leave it as is.